test/mix_unused/analyzers/unreachable/usages/helpers/aliases_test.exs

```diff
@@ -0,0 +1,55 @@
+defmodule MixUnused.Analyzers.Unreachable.Usages.Helpers.AliasesTest do
+  @moduledoc """
+  """
+  use ExUnit.Case
+
+  alias MixUnused.Analyzers.Unreachable.Usages.Helpers.Aliases
+
+  test "it is able to resolve simple aliases" do
+    aliases = Aliases.new(Code.string_to_quoted!(~s[
+      defmodule A.B.C do
+        alias A
+        alias A.B
+        alias A.B.C
+      end
+    ]))
+
+    assert Aliases.resolve(aliases, [:A]) == A
+    assert Aliases.resolve(aliases, [:B]) == A.B
+    assert Aliases.resolve(aliases, [:C]) == A.B.C
```

Comment:
I'm not sure, what happens with `Aliases.resolve(aliases, [:C, :D])`?

Reply:
it resolves `C` to `A.B.C`, returning `A.B.C.D`